Lavinia-api/Utilities/ModelBuilder.cs

```diff
@@ -87,9 +89,19 @@ public static List<County> BuildCounties(List<ResultFormat> results, IEnumerable
             Dictionary<string, County> countyModels = new Dictionary<string, County>();
             foreach (ResultFormat resultFormat in results)
             {
+                CountyDataFormat curCountyData = new CountyDataFormat();
+
                 if (!countyModels.ContainsKey(resultFormat.Fylkenavn))
```

Comment:
This could probably be done in an [extension method](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods), increasing readability and maintaining the current level of abstraction (not really necessary, but much cleaner).

Reply:
Not entirely sure how you were intending to use an extension method in this method, but I have tidied up the method somewhat.